specification/ai/HealthInsights/HealthInsights.RadiologyInsights/tspconfig.yaml

```diff
@@ -29,8 +29,7 @@ options:
     package-mode: "dataplane"
     package-version: 1.0.0
     package-dir: "azure-healthinsights-radiologyinsights"
-    package-name: "{package-dir}"
-    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
```

Comment:
removing instances of `emitter-output-dir` since `tsp-client` handles this for us now